model/src/main/kotlin/utils/LicenseResolver.kt

```diff
@@ -79,6 +79,7 @@ internal class LicenseResolver(
             }
 
             val curatedLicenseFindings = curationMatcher.applyAll(rawLicenseFindings, curations)
+                .mapNotNull { it.curatedFinding }.toSet()
```

Comment:
Nit: Could be `.mapNotNullTo(mutableSetOf()) { it.curatedFinding }` to avoid the creation of a temporary list.

Reply:
Done.